www/gatsby-config.js

```diff
@@ -345,6 +351,12 @@ module.exports = {
         nodeTypes: [`StartersYaml`],
       },
     },
+    {
+      resolve: `gatsby-plugin-i18n`,
```

Comment:
This is a nitpick so feel free to ignore:
Can you add a comment here that this is a local plugin. I can already see me looking up the "remote" plugin instead of the local one in some months.